README.rst

```diff
@@ -6,363 +6,106 @@ Additional modules for the Cyclus nuclear fuel cycle simulator from the
 University of Wisconsin - Madison are intended to be support innovative
 fuel cycle simulations with the Cyclus fuel cycle simulator.
 
-To see user and developer documentation for this code, please visit
-the `Cyclus Homepage`_.
+Cycamore require Cyclus. If you don't have installed Cyclus yet please checkout
+the `Cyclus website <http://fuelcycle.org/>`_ or the `Cyclus repository <https://github.com/cyclus/cyclus>`_.
 
-********************************
-Building and Installing Cycamore
-********************************
+Third party modules can also be installed (or developed) with additional
```

Comment:
Perhaps you can add something like this before this sentence: "Although Cycamore is meant to enable a basic fuel cycle simulation, you may want more functionality than it offers."

I think it's also discussed on the website more but I don't think it hurts to have a little extra explanation.